langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/internal/AbstractBedrockChatModel.java

```diff
@@ -127,6 +129,13 @@ protected InvokeModelResponse invoke(final InvokeModelRequest invokeModelRequest
         return getClient().invokeModel(invokeModelRequest);
     }
 
+    public BedrockRuntimeClient getClient() {
+        if (client == null) {
+            client = initClient();
```

Comment:
This is not a proper way to do lazy initialization, there might be problems in concurrent scenarios. Is lazy initialization required at all? If not, I would remove lazy init logic. If yes, I would re-implement this taking concurrency into consideration.

Reply:
fixed

by the way, thast's why i hate lombok. not convinent for this things